### PR TITLE
TSK-1925: clean up claimTaskAccTest

### PR DESCRIPTION
**Pull request by ryzheboka**
https://sonarcloud.io/summary/new_code?id=ryzheboka_taskana&branch=TSK-1925

---
Release Notes:

```

```
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java [hunk 1]
```diff
@@ -137,6 +160,17 @@ void should_ForceCompleteTask_When_TaskIsNotClaimed() throws Exception {
     assertTaskIsComplete(before, completedTask);
   }
 
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_CompleteTaskUsingForceComplete_When_TaskIsReadyForReview() throws Exception {
```
Comment (norman-schmidt) [inline lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java:165]: consistent naming

```suggestion
  void should_ForceCompleteTask_When_TaskIsReadyForReview() throws Exception {
```

**Review by norman-schmidt — CHANGES_REQUESTED** [review]
In addition to the other comments:
It would be good to keep consistent naming and rename some tests in CompleteTaskAccTest.java.

E. g.: 
should_ForceCompleteClaimedTask_When_TaskCallerIsNotTheOwner
to
should_ForceCompleteTask_When_ClaimedByAnotherUser

and so on :)

File: lib/taskana-core/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java [hunk 2]
```diff
@@ -229,149 +223,4 @@ void should_ForceCancelClaimTask_When_InReviewByAnotherUser() throws Exception {
...
-    assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getCompleted()).isNotNull();
-    assertThat(completedTask.getModified()).isEqualTo(completedTask.getCompleted());
-    assertThat(completedTask.isRead()).isTrue();
-    assertThat(completedTask.getOwner()).isEqualTo("user-1-2");
-  }
-
-  @WithAccessId(user = "admin")
-  @Test
-  void should_ReturnExceptions_WhenBulkDeletingInvalidTasks() throws Exception {
```
[the remark this answers is not in the document]
Comment (ryzheboka) [reply lib/taskana-core/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java:363]: It's testBulkDeleteTasksWithException in DeleteTaskAccTest

File: lib/taskana-core/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java [hunk 3]
```diff
@@ -229,149 +223,4 @@ void should_ForceCancelClaimTask_When_InReviewByAnotherUser() throws Exception {
...
-    taskService.completeTask(claimedTask.getId());
-
-    Task completedTask = taskService.getTask("TKI:400000000000000000000000000000000028");
-    assertThat(completedTask).isNotNull();
-    assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getCompleted()).isNotNull();
-    assertThat(completedTask.getModified()).isEqualTo(completedTask.getCompleted());
-    assertThat(completedTask.getCompleted().isAfter(before)).isTrue();
-    assertThat(completedTask.getModified().isAfter(before)).isTrue();
-    assertThat(completedTask.isRead()).isTrue();
```
[the remark this answers is not in the document]
Comment (ryzheboka) [reply lib/taskana-core/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java:269]: This assertion should be mainly relevant for "claim" tests. After claiming, a Task has automatically been read. That's probably why it's in this test. I will still add it to complete tests because forceComplete can be done from the READY state, so that the Task hasn't been read yet.

File: lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java [hunk 4]
```diff
@@ -191,6 +225,15 @@ void should_ThrowException_When_TaskCallerIsNotTheOwner() throws Exception {
     assertThat(e.getTaskId()).isEqualTo(task.getId());
   }
 
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_CompletingTaskInReviewByAnotherUser() throws Exception {
+    Task claimedTask = createTaskClaimedByUser_1_1().owner("other").buildAndStore(taskService);
```
[the remark this answers is not in the document]
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java:231]: Ok :)

File: lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java [hunk 5]
```diff
@@ -137,6 +160,17 @@ void should_ForceCompleteTask_When_TaskIsNotClaimed() throws Exception {
     assertTaskIsComplete(before, completedTask);
   }
 
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_CompleteTaskUsingForceComplete_When_TaskIsReadyForReview() throws Exception {
+    Instant before = Instant.now().truncatedTo(ChronoUnit.MILLIS);
+    Task claimedTask = createTaskClaimedByUser_1_1().buildAndStore(taskService);
```
[the remark this answers is not in the document]
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java:167]: Ok :)

File: lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java (hunk 1 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java:165]: Sounds better

**Review by norman-schmidt — APPROVED** [review]
